estuary/api/v1.py

```diff
@@ -274,3 +274,35 @@ def get_siblings(resource, uid):
         serialized_results.append(serialized_node)
 
     return jsonify(serialized_results)
+
+
+@api_v1.route('/relationships/<resource>/<uid>')
+def get_artifact_relationships(resource, uid):
+    """
+    Get one-to-many relationships of a particular artifact.
+
+    :param str resource: a resource name that maps to a neomodel class
+    :param str uid: the value of the UniqueIdProperty to query with
+    :return: a Flask JSON response
+    :rtype: flask.Response
+    :raises NotFound: if the item is not found
+    :raises ValidationError: if an invalid resource was requested
+    """
+    item = get_neo4j_node(resource, uid)
+    if not item:
+        raise NotFound('This item does not exist')
+
+    # relationship to be expanded
+    rel_name = request.args.get('rel_name')
```

Comment:
Since this is a required argument, it may be best to just put it in the route like:
```
/relationships/<resource>/<uid>/<relationship>
```